frame/contracts/src/tests.rs

```diff
@@ -1611,13 +1611,33 @@ fn lazy_removal_works() {
 		assert_matches!(child::get(trie, &[99]), Some(42));
 
 		// Run the lazy removal
-		Contracts::on_initialize(Weight::max_value());
+		Contracts::on_idle(Weight::max_value(), Weight::max_value());
```

Comment:
Isn't the first argument the block number and not some `Weight`? I think it was wrong before (passing a block number and thinking it was `Weight`).

Reply:
I changed the parameter to `System::block_number()`.